projects/sport-data-valley/seeds/questionnaires/daily_questionnaire_rheumatism.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+db_title = ''
+db_name1 = 'Dagelijkse vragenlijst reuma'
```

Comment:
all the other questionnaires have a snake_case name: e.g. `daily_questionnaire_zondag`

Reply:
We don't actually use the name anywhere. On base platform we use the key. The original idea was that the key has to be in filename format and could be symbolized, so only lowercase underscores and digits. Typically identical to the file name of the questionnaire to avoid confusion.

The name of a questionnaire was supposed to be a human readable description of a questionnaire, and only internally visible, e.g., to the researchers when previewing the questionnaire or when doing an export for this questionnaire.

Also I checked, and on SDV we use the questionnaire key to try and translate the name of hte questionnaire in the questionnaire manager interface.

But I do agree with your point that we should be consistent here so I'll change it.

Reply:
Fixed.